### ISSUE TITLE
bin/composer not working under Win10

### ISSUE THREAD

**Issue by LazerTiberius** [mapbender/mapbender#717]
# Bug:
Calling bin/composer under windows 10 results in 
```
bin/composer: line 1: ../../composer.phar: No such file or directory
```
Calling composer.phar directly works perfectly fine

# Steps to reproduce:
* Use windows 10 and git bash for windows
* Clone repo mapbender-starter
* cd into /application
* Call bin/composer

# Expected result:
Default help output from composer

**Comment by AxxL** [comment]
Does php.exe composer.phar work?

**Comment by LazerTiberius** [comment]
``` php.exe bin/composer ``` will just echo ```../../composer .phar ``` in to the bash

**Comment by AxxL** [comment]
See here: http://doc.mapbender.org/de/book/installation/installation_windows.html

php composer.phar install -d "C:\srvc\mapbender-starter\application" works. If:

- php is in path (otherwise something like C:/srv/php/php.exe)
- the corresponding PHP extensions are enabled

The remaining question is why the composer.phar has wandered into the root folder and is not besides the composer.json. Documentation has to be altered therefore.

The bin/composer is only for linux. we can provide a composer.bat for Windows if you like.

**Comment by LazerTiberius** [comment]
> The bin/composer is only for linux. we can provide a composer.bat for Windows if you like.

This does not make any sense, there is absolute no reason to keep this file under linux too.  Why you should call a wierd link if you can call the binary directly. Just adjust the bootstrap linux script and delete this file!

**Comment by AxxL** [comment]
I guess it's because composer.phar is in mapbender-starter whereas composer.json is in application. 

I don't see why composer.phar shouldn't be in application. It might then be shipped with mapbender-release, but I'm fine with it.

So I'm going with:

- moving composer.phar to the right destination
- get rid of bin/composer symlink
- adjust all neccessary bootstrap scripts
- adjust contributing.md
- adjust user-documentation (installation)

**Comment by werrolf** [comment]
The bin/composer is not "only for linux".

We will continue shipping composer because
# Ubuntu is still stuck on 1.0.0 beta2 from the darkest ages of PHP
# to make working on Windows a little less troublesome (similar motivation exists for providing a precompiled sassc via required package), in absence of a "Windows extras" optional package

I can only assume the composer symlink vs composer.phar dichotomy was a quick workaround for an incomplete .gitignore setup. This is fixed in [Starter pull #72](https://github.com/mapbender/mapbender-starter/pull/72), which only keeps bin/composer and updates the .gitignore on application/bin to keep composer (self-)updatable.

**Comment by eschuettler** [comment]
In order to get a better overview we will clean up old issues.
If the issue requires further attention, feel free to open a new issue.
Thank you!
